examples/platforms/efr32mg12/radio.c

```diff
@@ -518,8 +519,10 @@ otError otPlatRadioTransmit(otInstance *aInstance, otRadioFrame *aFrame)
         sCurrentBandConfig = config;
     }
 
-    RAIL_WriteTxFifo(gRailHandle, (uint8_t *)&aFrame->mLength, sizeof(aFrame->mLength), true);
-    RAIL_WriteTxFifo(gRailHandle, aFrame->mPsdu, aFrame->mLength - 2, false);
+    otEXPECT(aFrame->mLength >= IEEE802154_MIN_LENGTH && aFrame->mLength <= IEEE802154_MAX_LENGTH);
+    frameLength = (uint8_t)aFrame->mLength;
```

Comment:
mFrame->mLength has changed from a uint8_t to a uint16_t in commit ce26bffb, so we convert it back to a uint8_t here and write a uint8_t to the Tx Fifo.   Added an assert prior to this to check for writes > 127.